docs/helm-chart-checks.md

```diff
@@ -131,18 +131,34 @@ This section provides help on the basic usage of Helm chart checks with the podm
 
 ### Cluster Config
 
-Configuration of the `chart-testing` check can be achieved through the `--set` command line option, which in addition of configuring `chart-verifier` itself, can set check's configuration values.
+Configuration of the `chart-testing` check can be achieved in one of two ways: 
+
+1. Through the `--set` command line option, which in addition of configuring `chart-verifier` itself, can set check's configuration values.
     ```text
     $ chart-verifier                                                 \
         verify                                                       \
         --enable chart-testing                                       \
-        --set chartTesting.buildId=${BUILD_ID}                       \
-        --set chartTesting.upgrade=true                              \
-        --set chartTesting.skipMissingValues=true                    \
-        --set chartTesting.namespace=${NAMESPACE}                    \
-        --set chartTesting.releaseLabel="app.kubernetes.io/instance" \
+        --set chart-testing.buildId=${BUILD_ID}                       \
+        --set chart-testing.upgrade=true                              \
+        --set chart-testing.skipMissingValues=true                    \
+        --set chart-testing.namespace=${NAMESPACE}                    \
+        --set chart-testing.releaseLabel="app.kubernetes.io/instance" \
         some-chart.tgz
     ```
+1. Through the `--set-values` command line option, where a YAML file can be used to configure the check. For example for a config.yaml file content:
```

Comment:
Suggestion: Place this under the 1st step as the 2nd option (as a bullet item).

Reply:
Suggestion: "Use a YAML file similar to the following example and from the the `--set-values` CLI option, configure the `chart-testing` check:

Example YAML output"

Reply:
I finished up with:
* Option 2: Create a YAML file (config.yaml) similar to the following example:

and later:

Specify the file using the `--set-values` command line option:

Think it all looks a lot cleaner now.

Reply:
> I finished up with:
> 
> * Option 2: Create a YAML file (config.yaml) similar to the following example:
> 

When we say perform any one of the following steps, it implies that these are options. No need to call it out as Option 1 and 2. Remove Option 2 and retain the bullet item and the sentence.

Reply:
> Specify the file using the `--set-values` command line option:

Question:
- Is this step common to both options 1 and 2?

Sugegstion:
If this step is only for option 2, and this is indeed a verification step for the chart-testing check configuration, then I suggest to have a lead line saying:

-  Perform the following steps:

1. Create a YAML file (config.yaml) similar to the following example:
Note: Wait for the file to persist. After the file is persisted, you can verify a chart for the 'chart-testing' check configuration.
2. To verify a chart, specify the `config.yaml` file using the `--set-values` option in the Red Hat Helm chart-verifier CLI: